src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Offender.kt

```diff
@@ -43,6 +43,8 @@ data class Offender(
 
   val crd: LocalDate? = null,
 
+  val caseReferenceNumber: String? = null,
```

Comment:
style guides suggest this should generally be referred to as `crn`. 
It's how probation managers refer to it

Reply:
https://design-patterns.service.justice.gov.uk/content-standards/style-guide/

Reply:
Sure will update.